swagger-spec/swagger.yaml

```diff
@@ -349,6 +349,12 @@ tags:
           + `email_contact`
 
 
+      `v2.7`
```

Comment:
I'll add the changelog for versions 2.5 and 2.6 before this PR goes in so it doesn't look like we skipped over a few versions...